src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ReportsTest.kt

```diff
@@ -116,18 +116,72 @@ class Cas2ReportsTest : IntegrationTestBase() {
       val event2Id = UUID.randomUUID()
       val event3Id = UUID.randomUUID()
 
-      val old = Instant.now().minusSeconds(daysInSeconds(365))
-      val newer = Instant.now().minusSeconds(daysInSeconds(100))
-      val tooOld = Instant.now().minusSeconds(daysInSeconds(366))
+      val oldSubmitted = Instant.now().minusSeconds(daysInSeconds(365))
+      val oldCreated = oldSubmitted.minusSeconds(daysInSeconds(7))
```

Comment:
Assumed application creation dates are 7 days before submission dates.

Reply:
this makes sense - could be any date in the past for the purposes of testing.
Minor change but I wonder if this commit could be be fixedup into the integration test commit before it?

Reply:
Squashed and pushed these two commits - nice suggestion